Ej2-Documentation(846715)- UG documentation for Row template

Gayathri4135 opened the pull request:
Modified the row template

SyncfusionBuild commented:
**Build Status: INQUEUE :clock3:<br/>Build Location: [Ej2-846715-RowTemplate](https://jenkins.syncfusion.com/job/Content/job/ej2-grid-docs/317/console)**

SyncfusionBuild commented:
**Build Status: INPROGRESS :arrows_clockwise:<br/>Build Location: [Ej2-846715-RowTemplate](https://jenkins.syncfusion.com/job/Content/job/ej2-grid-docs/317/console)**

SyncfusionBuild commented:
**CI Status: SUCCESS :white_check_mark:<br/>Source Compilation: SUCCESS :white_check_mark:<br/>Build Location: [Ej2-846715-RowTemplate](https://jenkins.syncfusion.com/job/Content/job/ej2-grid-docs/317/)<br/>**

BalajiSGrid commented:
Technical review has been completed

Gayathri4135 commented:
Changes reviewed in this PR: https://github.com/syncfusion-content/ej2-grid-docs/pull/36
The reviewed changes :
1. Remove the unnecessary properties.
Removed the unnecessary properties.
2. Change the content of limitation.
Changed the content of limitation.
3. Update the version in the sample.
Updated the version in the sample
hotfix version 23.1.36: https://github.com/syncfusion-content/ej2-grid-docs/pull/76

SyncfusionBuild commented:
**Build Status: INPROGRESS :arrows_clockwise:<br/>Build Location: [Ej2-846715-RowTemplate](https://jenkins.syncfusion.com/job/Content/job/ej2-grid-docs/353/console)**

MaithiliyK reviewed (approved):
changes are fine
